From 5472d8aff733a89bf458f4c393a770cf8e7f0423 Mon Sep 17 00:00:00 2001
From: Mark D Horn <mark.d.horn@intel.com>
Date: Thu, 3 Dec 2020 17:48:16 -0800
Subject: [PATCH 1/4] Makefile: Set DefaultsPath for stateless

Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>
Signed-off-by: Mark D Horn <mark.d.horn@intel.com>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 19f8052..5dea5a5 100644
--- a/Makefile
+++ b/Makefile
@@ -101,7 +101,8 @@ BASE_LDFLAGS = ${SHRINKFLAGS} \
 	-X ${PROJECT}/internal/pkg/criocli.DefaultsPath=${DEFAULTS_PATH} \
 	-X ${PROJECT}/internal/version.buildDate=${BUILD_DATE} \
 	-X ${PROJECT}/internal/version.gitCommit=${COMMIT_NO} \
-	-X ${PROJECT}/internal/version.gitTreeState=${GIT_TREE_STATE}
+	-X ${PROJECT}/internal/version.gitTreeState=${GIT_TREE_STATE} \
+	-X ${PROJECT}/internal/pkg/criocli.DefaultsPath=/usr/share/defaults/crio
 
 GO_LDFLAGS = -ldflags '${BASE_LDFLAGS} ${EXTRA_LDFLAGS}'
 
-- 
2.35.1

